File: 8-kyu/Find the Difference in Age between Oldest and Youngest Family Members/find-the-difference-in-age-between-oldest-and-youngest-family-members.py
# Find the Difference in Age between Oldest and Youngest Family Members
# https://www.codewars.com/kata/5720a1cb65a504fdff0003e2/solutions/javascript

def difference_in_ages(ages):
  return (min(ages), max(ages), max(ages) - min(ages))

File: 8-kyu/Find the Difference in Age between Oldest and Youngest Family Members/find-the-difference-in-age-between-oldest-and-youngest-family-members.js
// Name
/*
  Find the Difference in Age between Oldest and Youngest Family Members
*/
// Url
/*
  https://www.codewars.com/kata/5720a1cb65a504fdff0003e2/train/javascript
*/

function differenceInAges(ages) {
  const min = Math.min(...ages);
  const max = Math.max(...ages);

  return [min, max, max - min];
}
